package.json

```diff
@@ -90,7 +90,7 @@
   "homepage": "https://github.com/ReactiveX/RxJS",
   "devDependencies": {
     "benchmark": "1.0.0",
-    "benchpress": "2.0.0-alpha.37.2",
+    "benchpress": "^2.0.0-beta.1",
```

Comment:
I'll close and reopen PR, I screwed up too many things here. Linked to wrong issue in commit message too. :cry: